lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1018,6 +1017,7 @@ fc_extras
     import iris.exceptions
     import iris.std_names
     import iris.util
+    from iris._lazy_data import as_lazy_data
```

Comment:
Does it matter what order these are in? I just noticed that in the other files the private modules are imported first.